### PR TITLE
Instructions for lldb use on linux

### PR DESCRIPTION
**Pull request by lindsayad**
Details compiler flags required and lldb installation setup
needed to get effective lldb debugging on linux (or at least Ubuntu)

### CODE REVIEW

**Comment by lindsayad** [comment]
@rwcarlsen This is largely Ubuntu driven, but maybe there's some stuff in here that might be interesting to you, since I know you use or are interested in using clang/lldb on linux.

File: modules/doc/content/getting_started/installation/manual_installation_linux_lldb.md [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+[is known to have issues](https://bugzilla.redhat.com/show_bug.cgi?id=1740927)
+reading debug info from `libstdc++`. The most straightforward fix to this is to
+have `clang` institute its own debug information:
+
+!package! code
+export CFLAGS='-fstandalone-debug'
+export CXXFLAGS='-fstandalone-debug'
+!package-end!
+
+Note that we do not pass this option into `CPPFLAGS` because `gfortran` doesn't
```
Comment (dschwen) [inline modules/doc/content/getting_started/installation/manual_installation_linux_lldb.md]: Us pass it into `CXXFLAGS` above

Comment (lindsayad) [reply modules/doc/content/getting_started/installation/manual_installation_linux_lldb.md]: Huh, yea that is intentional. We explicitly pass it to the C compiler with `CFLAGS` and to the CXX compiler with `CXXFLAGS`. But `CPPFLAGS` is used by the C, CXX, and fortran  compilers, so if we don't want the fortran compiler compiling with that flag, then we make sure its not in `CPPFLAGS`

Comment (dschwen) [reply modules/doc/content/getting_started/installation/manual_installation_linux_lldb.md]: D'uh, "...do **not** pass..."

**Comment by milljm** [comment]
Sure, I'll accept this. But it makes me wonder if we should have a dedicated 'miscellaneous how-to' like section rather than stamp this on our main Getting Started link. Well done figuring it out!

**Review by milljm — APPROVED** [review]
Everything looks good except perhaps the location of this link. But we can decide this later.

**Pull request merged**